build-system/server/new-server/transforms/utilities/cdn.ts

```diff
@@ -55,7 +55,7 @@ export function CDNURLToLocalDistURL(
   useMaxNames = false,
 ): URL {
   url.protocol = 'http';
-  url.hostname = 'localhost';
+  url.hostname = '0.0.0.0';
```

Comment:
@jridgewell is this correct?

Reply:
also question does anyone know if 0.0.0.0 works on windows?

/cc @rsimha

Reply:
No, this isn't right. If i request the page from my phone, `0.0.0.0` is going to tell me to look up this URL on my phone server, which doesn't exist. You need to know the host of the page is currently being requested, and rewrite the URL relative to that host.

Reply:
We run integration tests on Windows, so you can look at logs from older CI builds. [This build](https://github.com/ampproject/amphtml/pull/33423/checks?check_run_id=2169845815#step:4:43) suggests that Windows doesn't use 0.0.0.0.

![image](https://user-images.githubusercontent.com/26553114/112067671-cf795280-8b3e-11eb-8700-1b2a4e80dbbe.png)